assets/js/base/components/cart-checkout/form-step/style.scss

```diff
@@ -14,6 +14,10 @@
 	}
 }
 
+.wc-block-components-checkout-step--disabled {
```

Comment:
With this new selector, do you think we can remove the one in [line 29](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/trunk/assets/js/base/components/cart-checkout/form-step/style.scss#L25)?

Reply:
You're right - it's no longer needed. Removed.